docs/documentation/make-first-prototype/branching.md

```diff
@@ -0,0 +1,78 @@
+# Branching (showing pages depending on user input)
+
+Our first question asks the user how many balls they can juggle. We’re going to send them to an ‘ineligible’ page if they can only juggle 2 or less.
+
+To do this, we’re going to need an ‘ineligible.html’ page, and some logic to decide when to send them there.
+
+This kind of logic goes in a file called `app/routes.js`. Routes tell the kit what to do when the user goes to specific pages, and it’s written in JavaScript.
+
+The route takes the answer the user gave to the first question and either shows them the next question or the ineligible page, depending on their answer.
+
+## Create the ineligible page
+
+1. Make an `ineligible.html` page by copying `template-content-page.html`
```

Comment:
```suggestion
1. Make an `ineligible.html` page by copying `template-content-page.html`.
```

Reply:
Maybe the user does not need to know this, but I wondered - how does copying `template-content-page.html` make an `ineligible.html` page? Can we add content here about copying, or else link back to relevant content elsewhere in the Kit docs?

Also, what's the difference between 'creating' an ineligible page (as in the title) and 'making' one (as in step 1)?